src/components/history/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Table } from 'reactstrap';
+import { Updates } from '../../types';
+import {
+  TableHeader,
+  TableRow,
+  TableHeaderCell,
+  TableBody,
+  TableCell,
+  HistoryTitle
+} from './styled';
+
+interface HistoryProps {
+  conceptId: string;
+  resourceId?: string;
+}
+
+// eslint-disable-next-line no-empty-pattern
+const History = ({}: HistoryProps) => {
+  const updates: Updates[] = [
+    {
+      conceptId: '1',
+      resourceId: '1',
+      id: '1',
+      person: 'John Doe',
+      datetime: '2022-02-26T10:00:00Z',
+      operations: 'Added a new concept'
+    },
+    {
+      conceptId: '2',
+      resourceId: '2',
+      id: '2',
+      person: 'Jane Smith',
+      datetime: '2022-02-25T11:00:00Z',
+      operations: 'Updated related terms of the concept'
+    },
+    {
+      conceptId: '3',
+      resourceId: '3',
+      id: '3',
+      person: 'Bob Johnson',
+      datetime: '2022-02-24T12:00:00Z',
+      operations: 'Added a new attribute to the concept'
+    }
+  ];
+  // const [updates, setUpdates] = useState<Updates[]>([]);
+  // const [updates, setUpdates] = useState<Updates[]>(dummyData);
+
+  /*   useEffect(() => {
+    const fetchUpdates = async () => {
+      try {
+        const response = await getUpdateByConceptId(conceptId);
+        setUpdates(response.data);
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error(error);
+      }
+    };
+    fetchUpdates();
+  }, [conceptId]); */
```

Comment:
```suggestion
```

Reply:
Ikke fan av utkommentert kode og TODO i commits, litt for ofte at det bare blir liggende. Bedre å lage issues elns og lime inn en eventuell kodesnutt der

Reply:
+1